Lib/reprlib.py

```diff
@@ -36,6 +36,17 @@ def wrapper(self):
     return decorating_function
 
 class Repr:
+    _lookup = {
+        "repr_tuple": ("builtins", "tuple"),
+        "repr_list": ("builtins", "list"),
+        "repr_array": ("array", "array"),
+        "repr_set": ("builtins", "set"),
+        "repr_frozenset": ("builtins", "frozenset"),
+        "repr_deque": ("collections", "deque"),
+        "repr_dict": ("builtins", "dict"),
+        "repr_str": ("builtins", "str"),
+        "repr_int": ("builtins", "int"),
```

Comment:
There is redundancy here. The type name is repeated twice. Also, "repr_" is the constant prefix. You could just map the type name to module name, e.g. `'deque': 'collections'`.

BTW, use single quotes. This is more preferable style in general and in this module in particular.